### DISCUSSION TITLE
Collaboration Questions

### DISCUSSION THREAD

**Discussion by spikespaz** [d86leader/wzmach#7] [General]
I am creating this thread for general questions whose answers will help me or others contribute.

**Comment by spikespaz** [comment]
What's the idea behind having separate X11 and Wayland configurations? Is it for computers with multiple user/desktop environments?

If it is to handle differences in keybinds that are intended to interact differently depending on the current active environment, would it perhaps be a good idea to come up with another solution that can incorporate any combination of display server and desktop environment?

**Reply by maurges** [reply]
One example of why I use separate configurations for x11 and wayland is already provided in the default config: KDE on wayland has built-in four finger gestures that I would like to use, but KDE for x11 doesn't. I myself still switch between the two, so it's useful for me.

I think any additional flexibility is not necessary now, it would make things more complicated. And in any case, the users can load their own config via the command line options, so they can build their own solution with that.
